### ISSUE TITLE
the value of  " i " from tf.train.range_input_producer(epoch_size, shuffle=False).dequeue()

### ISSUE THREAD

**Issue by wangpeng138375** [tensorflow/models#794]
## Please let us know which model this issue is about (specify the top-level directory)
the ptb language model  , in the reader.py 
`i = tf.train.range_input_producer(epoch_size, shuffle=False).dequeue()
    x = tf.strided_slice(data, [0, i * num_steps],
                         [batch_size, (i + 1) * num_steps])
    x.set_shape([batch_size, num_steps])
    y = tf.strided_slice(data, [0, i * num_steps + 1],
                         [batch_size, (i + 1) * num_steps + 1])`
every time when uses ' i ' , it comes from the queue newly , so all of ' i ' above are not the same ?!

**Comment by michaelisard** [comment]
Tensorflow usage questions are better asked on [Stackoverflow](http://stackoverflow.com/questions/tagged/tensorflow). Github issues are for bug reports and installation problems.
